crates/stackable-operator/src/utils/cluster_info.rs

```diff
@@ -0,0 +1,34 @@
+use std::str::FromStr;
+
+use crate::commons::networking::DomainName;
+
+const KUBERNETES_CLUSTER_DOMAIN_DEFAULT: &str = "cluster.local";
+
+#[derive(Debug, Clone)]
+pub struct KubernetesClusterInfo {
+    pub cluster_domain: DomainName,
+}
+
+impl KubernetesClusterInfo {
+    pub fn new(cli_kubernetes_cluster_domain: &Option<DomainName>) -> Self {
```

Comment:
General nit: Always use `Option<&T>`, not `&Option<T>`. (`&mut Option` has uses, but `&Option` doesn't give you anything new.)

Reply:
Good catch, I actually did also see it but forget to change it :see_no_evil: